### PR TITLE
Full stack Web Development YT link added

### PR DESCRIPTION
**Pull request by khushbumaheshwarii**
## Issues

Closes Issue #761 

 
## Screenshots

image

### CODE REVIEW

**Review by navharsh — APPROVED** [review]
LGTM! 😄

**Review by pratyushmp — APPROVED** [review]

**Review by ujjawaltyagii — APPROVED** [review]
Lgtm 👍

**Comment by rupali-codes** [comment]
@khushbumaheshwarii could you please resolve merge conflicts?

**Comment by khushbumaheshwarii** [comment]
> @khushbumaheshwarii could you please resolve merge conflicts?
 i will do it soon give me some time

**Comment by rupali-codes** [comment]
> > @khushbumaheshwarii could you please resolve merge conflicts?
> > i will do it soon give me some time

Sure, also please reference the respective issue

**Comment by khushbumaheshwarii** [comment]
> > > @khushbumaheshwarii could you please resolve merge conflicts?
> > > i will do it soon give me some time
> 
> Sure, also please reference the respective issue

can you please review conflicts changes

File: database/youtube/web-development.json [hunk 1]
```diff
@@ -174,5 +174,13 @@
     "category": "youtube",
     "subcategory": "web-development",
     "language": "english"
+  },
+  {
+    "name": "CodeHelp",
+    "description": "This is one of the Best Free Beginner to Advance course for all those people who wants to learn Full Stack Web Development",
```
Comment (CBID2) [inline database/youtube/web-development.json:180]: [Proofreading]
```suggestion
    "description": "This is one of the best free beginner to advanced course for people who want to learn Full Stack Web Development",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @khushbumaheshwarii! :) I pointed out a proofreading error in my inline comment. In the future, consider running your descriptions through Grammarly or Spellcheck before making a pull request. Other than that, you're off to a great start! :)

**Review by CBID2 — APPROVED** [review]
LGTM!

File: database/youtube/web-development.json [hunk 2]
```diff
@@ -174,5 +174,13 @@
...
     "subcategory": "web-development",
     "language": "english"
+  },
+  {
+    "name": "CodeHelp",
+    "description": "This is one of the best free beginner to advanced course for people who want to learn Full Stack Web Development",
+    "url": "https://www.youtube.com/playlist?list=PLDzeHZWIZsTo0wSBcg4-NMIbC0L8evLrD",
+    "category": "youtube",
+    "subcategory": "web-development",
+    "language": "Hindi"
```
Comment (rupali-codes) [inline database/youtube/web-development.json:184]:
```suggestion
    "language": "hindi"
```

**Review by khushbumaheshwarii — COMMENTED** [review]
made changes

**Review by rupali-codes — APPROVED** [review]
lgtm

**Pull request merged**